src/sparql/icd11foundation-relevant-diseases.sparql

```diff
@@ -0,0 +1,32 @@
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+
+
+### All diseases
+SELECT DISTINCT ?term ?label ?deprecated
+WHERE {
+  { 
+    { 
+     ?s1 ?p1 ?term . 
+     ?term rdfs:subClassOf* <http://id.who.int/icd/entity/455013390> .
```

Comment:
## Relevant diseases SPARQL
I created this SPARQL query to select diseases, using `icd11.foundation:455013390` _(ICD Category)_ as the root.

### Additional information
I'm not 100% sure about this selection. But it appears that all of the diseases fall under it. However, I'm not sure that everything category that falls under it are disease categories, though most appear to be.

<img width="550" alt="icd11 foundation tree" src="https://github.com/monarch-initiative/mondo-ingest/assets/13045020/43377e13-7d40-4f5e-8c3e-8ab93b01f656">

### Questions
1. @sabrinatoro (CC @matentzn @twhetzel ) **Did I make the correct selection?**: Could you look at the above screenshot and the ontology itself and see if I selected the right thing? I uploaded to GoogleDrive: [`icd11foundation.owl`](url)
2. @sabrinatoro @matentzn @twhetzel **Exclusions**: It looks like for all the other ontologies, a single root term is sufficient for all diseases. If we select a root term but there are non-diseases under it, then I suppose we should start populating the exclusions TSV?
3. @matentzn **2 queries?**: You asked me to create 1 query for disease selection, but it looks like we want 2 queries for all ontologies, `relevant-diseases` and `relevant-signature`. From what I can see, for all/most files, we used the same root disease term for both queries. It appears that both queries select the same triples, but then differ in filtering and display.
4. @matentzn **FYI - incorrect comments in SPARQL queries**: For example, the MedGen query says "SNOMED". And then for many of the queries for several sources, it has the comment "### For Disease Ontology, we are interested in all terms". I'll likely make a PR for this.

Reply:
> Did I make the correct selection?

I would suggest:

1. Stick for now with your selection (icd11.foundation:455013390)
2. Make new issue to identify true disease roots and assign to @twhetzel 
3. @twhetzel then Open in Protege, figure out which roots are real diseases and update the SPARQL query later

> Exclusions: 

Dont worry about this in this PR. Keep empty table. Curation team will deal with this.

> 2 queries?

I dont remember what the disease one does and where it is used. I would just add relevant signature for now and see what breaks.

> FYI - incorrect comments in SPARQL queries

You can fix cosmetic things like this in this PR here. No need to open a new PR just to add a comment.

Reply:
[Sabrina wrote](https://github.com/monarch-initiative/mondo-ingest/pull/412#issuecomment-1954523828): 
> > Did I make the correct selection?
> 
> Yes, I think it is ok. It is possible that other branches have diseases, but it looks most of them are there. 
> 
> > Exclusions
> 
> Since ICD is not an actual ontology, we cannot trust that we can exclude one term and everything underneath this term unless we take a look at it first. 
> 
> I hope this helps!

Reply:
Remaining tasks here are for me (@joeflack4):
- [ ] **Disease roots**: Make new issue to identify true disease roots and assign to Trish
- [ ] Remove `icd11foundation-relevant-diseases.sparql`
- [ ] Fix cosmetic SPARQL comment issues

Reply:
All tasks in this thread done!